app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { comment: @comment } }
+    end
+  end
+
+  def comment_params
+    params
+      .require(:comment)
+      .permit(:text)
+      .merge(author: current_user, post_id: params.require(:post_id))
```

Comment:
- [Optional] As already mentioned above, you are exposing confidential information since this method is not made private

Reply:
@FrederickMih Thank you for your valuable suggestion. I have implemented the suggested changes https://github.com/NayanKaran/blog-using-rails/pull/8/commits/6c8b24eeef6e940e403004f42122e500d18023c7 .